_locales/uk/messages.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extensionDescription": {
+        "message": "Додати для закладок пункт меню 'Відкрити в контейнері'",
+        "description": "Short description of the extension."
+    },
+    "open_one_in_container_tab": {
+        "message": "Відкрити в контейнері",
```

Comment:
For the "Open in a New Tab" contextmenu option on a bookmark, Firefox uses the following ([source](https://hg.mozilla.org/l10n-central/uk/file/341c6fdc1be93cad42c0b48e2a1a495797a0aff0/browser/browser/places.ftl#l8)):
```
Відкрити в новій вкладці
```

For "Open Link in New Container Tab" (when you right-click on a link), Firefox uses the following ([source](https://hg.mozilla.org/l10n-central/uk/file/341c6fdc1be93cad42c0b48e2a1a495797a0aff0/browser/browser/browserContext.ftl#l104)):
```
Відкрити посилання в новій вкладці контейнера
```

Based on these (especially the first, combined with the patterns that I have observed from the second source), I would expect the following to be the most consistent translation:

```
Відкрити в новій вкладці контейнера
```

instead of `Відкрити в контейнері`, which is proposed in this pull request.
Could you double-check to see if the translation should be changed (similarly for the message in "open_all_in_container_tab")?

Reply:
Yep, the Ukrainian translation in the Firefox codebase is just as correct (but more verbose). Phrase `Відкрити в новій вкладці контейнера` is correct and will be consistent with that translation. Similar translation for "open_all_in_container_tab" is `Відкрити все в нових вкладках контейнера`.